django_freeradius/base/models.py

```diff
@@ -513,10 +513,14 @@ def __str__(self):
 
 
 class AbstractRadiusBatch(TimeStampedEditableModel):
+    '''
+    Abstract class for making mutliple users of
+    django-radius.
+    Note: You'll (most probably) need to add the following
+    field in your concrete class:
```

Comment:
simply add the unique here and get rid of this docstring, openwisp-radius will take care of doing the right thing